### DISCUSSION TITLE
［速报］马上就是10.1小长假了

### DISCUSSION THREAD

**Discussion by hutao48** [share121/inter-knot#5481] [其他]
明天就是10.1国庆节了，各位有没有什么安排喵
![1000009064](https://github.com/user-attachments/assets/7b5541f1-7df3-464c-b548-7f885204f07a)

**Comment by hutao48** [comment]
这里补充一下，本人准备大阪2日游，2号的机票

**Comment by aomengxin** [comment]
打算成都四日游，也是2号走

**Comment by March-777th** [comment]
只想在家宅着<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" >好累啊

**Comment by cuijiaqi0406** [comment]
被迫参加了夏令营，明天去<img src="https://bbs-static.miyoushe.com/static/2023/11/16/189d73f83e4892766ac8197f2df156d0_3183903856565806928.png" alt="比利-呜呜呜" width="75" >
